frontend/ui/data/css_tweaks.lua

```diff
@@ -134,6 +134,11 @@ h1 + h6, h2 + h6, h3 + h6, h4 + h6, h5 + h6 { page-break-before: avoid !importan
                     ]],
                 },
             },
+            {
+                id = "page-break-after-body";
+                title = _("New page on next <body>"),
+                css = [[body { page-break-after: always !important; }]],
```

Comment:
This isn't saying the same thing as the text though?

Reply:
The first body always starts with a new page, so we shouldn't force `page-break-before` once more.
But we need a page-break before the next body, so we force `page-break-after`.

Reply:
I don't understand why you would want that as a style tweak? That sounds like a problem with fb2.css. Or is it a workaround for a broken document that doesn't work with proper ones?